src/main/java/io/cdap/plugin/snowflake/source/batch/SnowflakeBatchSourceConfig.java

```diff
@@ -16,12 +16,12 @@
 
 package io.cdap.plugin.snowflake.source.batch;
 
+import com.google.common.base.Strings;
 import io.cdap.cdap.api.annotation.Description;
 import io.cdap.cdap.api.annotation.Macro;
 import io.cdap.cdap.api.annotation.Name;
 import io.cdap.cdap.etl.api.FailureCollector;
 import io.cdap.plugin.snowflake.common.BaseSnowflakeConfig;
-
```

Comment:
why this change?

Reply:
Reverted f1e33c02ce1b80b5471c0770c455fd65b3246e1a